client/src/app/site/login/services/browser-support.service.ts

```diff
@@ -0,0 +1,54 @@
+import { Injectable } from '@angular/core';
+
+import { DeviceDetectorService, DeviceInfo } from 'ngx-device-detector';
+
+enum SmallestSupportedBrowserVersion {
+    Chrome = 83,
```

Comment:
Are you really sure that v81 or v80 is already unsupported by OpenSlides?! It's very hard. I prefere to use a more tolerant way.

Reply:
I am not, I simply took the current version :smile: